HACKING.adoc

```diff
@@ -370,6 +385,7 @@ Using `opam install . --assume-built` will simply remove the
 package for the compiler, then run the installation instructions
 (`make install`) in the working directory, tracking the installed files
 correctly. The main difference with the `opam-custom-install` version
+
```

Comment:
```suggestion
```

Reply:
unrelated to the current change, but ok :-)